opm/input/eclipse/Schedule/UDQ/UDQActive.cpp

```diff
@@ -30,13 +30,24 @@
 #include <opm/input/eclipse/Units/UnitSystem.hpp>
 
 #include <algorithm>
+#include <cassert>
```

Comment:
see no asserts

Reply:
> see no asserts

Erm, yes, you're right.  There was one while I developed this, but I ended up not needing it.  I've pushed an update to remove this include statement.